utils/release_helper.py

```diff
@@ -0,0 +1,786 @@
+
+
+def get_latest_release(repo) -> object:
+    releases = get_repo_releases(repo)
+    # The API returns the list already ordered by published date.
+    return releases[0]
+
+
+# Repository Versions
+def extract_version_from_tag_name(version) -> str:
+    return version.split("v")[1]
+
+
+def get_latest_version(repo) -> str:
+    release = get_latest_release(repo)
+    return extract_version_from_tag_name(release.tag_name)
+
+
+# Repository Next Release
+def bump_version(current_version, bump_position) -> str:
+    # version format example: 0.1.64
+    if bump_position == 'minor':
+        position = 2
+    elif bump_position == 'major':
+        position = 1
+    else:
+        position = 0
+
+    split_version = current_version.split('.')
+    bumped = int(split_version[position]) + 1
+    split_version[position] = str(bumped)
+    return ".".join(split_version)
+
+
+def bump_master_version(old_version, new_version):
+    old_minor_version = old_version.split('.')[2]
+    new_minor_version = new_version.split('.')[2]
+    root_dir = os.path.dirname(os.path.dirname(__file__))
+    with open(os.path.join(root_dir, "CMakeLists.txt"), mode='r', encoding='utf-8') as file:
+        content = file.readlines()
+
+    for ln, line in enumerate(content):
+        if f'set(SSG_PATCH_VERSION {old_minor_version}' in line:
+            content[ln] = content[ln].replace(old_minor_version, new_minor_version)
+
+    with open(os.path.join(root_dir, "CMakeLists.txt"), mode='w', encoding='utf-8') as file:
+        file.writelines(content)
+
+
+def get_friday(date) -> datetime:
+    return date + timedelta((4-date.weekday()) % 7)
+
+
+def get_monday(date) -> datetime:
+    return date + timedelta((0-date.weekday()) % 7)
+
+
+def get_next_stabilization_date(release_date) -> datetime:
+    two_weeks_before = release_date - timedelta(weeks=2)
+    stabilization_monday = get_monday(two_weeks_before)
+    return stabilization_monday.date()
+
+
+def get_next_release_date(latest_release_date) -> datetime:
+    two_months_ahead = latest_release_date + timedelta(days=60)
+    return get_friday(two_months_ahead)
+
+
+def is_next_release_in_progress(repo) -> bool:
+    stabilization_branch = get_next_stabilization_branch(repo)
+    branches_names = [branch.name for branch in get_repo_branches(repo)]
+    if stabilization_branch in branches_names:
+        return True
+    else:
+        return False
+
+
+def get_next_release_state(repo) -> str:
+    if is_next_release_in_progress(repo):
+        return 'In Stabilization Phase'
+    else:
+        return 'Scheduled'
+
+
+def get_next_release_version(repo) -> str:
+    current_version = get_latest_version(repo)
+    next_version = bump_version(current_version, 'minor')
+    return next_version
+
+
+def get_next_stabilization_branch(repo) -> str:
+    next_release = get_next_release_version(repo)
+    return f'stabilization-v{next_release}'
+
+
+# Communication
+def get_communication_channels(phase='release') -> None:
```

Comment:
I would rather call it print_communication channels, because when used later, I was wondering what this function does when its result is not assigned to anything actually.

Reply:
Good. The initial function was refactored but not its name. Thanks. I agree and will rename it.

Reply:
Done.